vagrant/mesos-netmodules/mesos-setup.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+
```

Comment:
Can we create a vbox image with all this pre-installed?
Its perfectly fine to do that as a separate PR